src/main/java/org/mule/module/apikit/validation/body/form/transformation/MultipartBuilder.java

```diff
@@ -109,6 +109,13 @@ public Multipart build() throws InvalidFormParameterException {
         }
       }
 
+      for (Entry<String, Parameter> formParameter : formParameters.entrySet()) {
+        if (!parametersInPayload.contains(formParameter.getKey()) && formParameter.getValue().isRequired()
+            && formParameter.getValue().getDefaultValue() == null) {
```

Comment:
It would be great to add a test for the case in which the required parameter is not sent but it has a default value